.gitlab-ci.yml

```diff
@@ -191,14 +191,14 @@ build_on_incline:
     - if: '$CI_PROJECT_ROOT_NAMESPACE == "exasgd"'
 
 pnnl_cleanup:
-  stage: test
-  dependencies:
-    - build_on_incline
+  stage: .pre
```

Comment:
https://stackoverflow.com/questions/56324070/with-gitlab-ci-how-to-disable-repository-clone-for-one-job - let's remove the cloning step for this pipeline.